toolbox/fdc3-workbench/index.html

```diff
@@ -2,19 +2,20 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+		<link rel="icon" href="/favicon.ico" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<meta name="theme-color" content="#000000" />
 		<meta
 			name="description"
 			content="FDC3 workbench application, designed to help develop and test FDC3 integrations"
 		/>
-		<link rel="apple-touch-icon" href="%PUBLIC_URL%/fdc3-icon-192.png" />
-		<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+		<link rel="apple-touch-icon" href="/fdc3-icon-192.png" />
+		<link rel="manifest" href="/manifest.json" />
```

Comment:
```suggestion
		<link rel="apple-touch-icon" href="fdc3-icon-192.png" />
		<link rel="manifest" href="manifest.json" />
```
See previous comment